scribus/ui/alignselect.cpp

```diff
@@ -110,3 +110,7 @@ void AlignSelect::languageChange()
 	TextB->setToolTip( tr("Align Text Justified"));
 	TextF->setToolTip( tr("Align Text Forced Justified"));
 }
+void AlignSelect::changeIcon(QToolButton* button, QString iconName)
+{
+	button->setIcon(IconManager::instance()->loadIcon(iconName));
```

Comment:
Since this one line function is called only twice, why don’t you just call this line directly?